modules/uhsg_search/uhsg_search.module

```diff
@@ -30,12 +30,17 @@ function uhsg_search_preprocess_node(&$variables) {
   if ($variables['view_mode'] == 'constrained') {
     $node = $variables['node'];
     if ($node->getType() == 'article') {
-      $variables['attributes']['data-type'] = uhsg_search_article_is_degree_programme_specific($node) ? 'article_degree_programme_specific' : 'article_general';
+      $variables['attributes']['data-type'] = [];
+      $variables['attributes']['data-type'][] = uhsg_search_article_is_degree_programme_specific($node) ? 'article_degree_programme_specific' : 'article_general';
+      if (uhsg_search_article_is_other_education_provider_specific($node)) {
+        $variables['attributes']['data-type'][] = 'article_other_education_provider_specific';
+      }
```

Comment:
This makes the filters work weirdly?
![Clicking filters](https://user-images.githubusercontent.com/93661/54355684-6236ab00-4662-11e9-8fb6-b80a647cdd64.gif)

Reply:
Pushed: Do not list other education specific article as general